pages/checkout.vue

```diff
@@ -123,45 +120,45 @@ import {
   SfButton,
   SfPersonalDetails,
   SfShipping,
-  SfPayment,
   SfConfirmOrder,
   SfOrderReview,
   SfCheckbox,
   SfInput,
 } from "@storefront-ui/vue"
 import { useAsync } from "@nuxtjs/composition-api"
-import { useCheckout, useCart, useUiState } from "@/composables"
-import { useNuxtApp } from "#app"
+import { useCheckout, useCart, useUiState, usePaymentMethods, useUser } from "@/composables"
+import { computed, ref, useNuxtApp } from "#app"
+import { buildPaymentMethodInput, defaultPaymentDetails } from "@/composables/helpers"
 
 export default {
   name: "Checkout",
   components: {
     SfSteps,
     SfPersonalDetails,
     SfShipping,
-    SfPayment,
     SfConfirmOrder,
     SfOrderReview,
     SfButton,
     SfCheckbox,
     SfInput,
   },
-  setup() {
+  setup(_prop, context) {
     const nuxt = useNuxtApp()
     const countries = nuxt.nuxt2Context.$config.countries
+    const { locale } = context.root.$i18n
+    const currencyCode = context.root.$i18n.getNumberFormat(locale)?.currency?.currency
+
     const currentStep = ref(0)
     const { cart } = useCart()
     const { checkout, loadFromCart, setPersonalInfo, setShippingInfo } = useCheckout()
     const { toggleLoginModal } = useUiState()
     const { createAccountAndLogin } = useUser()
+    const { tokenizeCard, addPaymentMethodByTokenizeCard } = usePaymentMethods()
     const showCreateAccount = ref(false)
     const password = ref(null)
     const confirmPassword = ref(null)
     const transition = "sf-fade"
 
-    const months = []
-    const years = []
-
     enum Steps {
       GO_TO_SHIPPING = "Go to Shipping",
```

Comment:
add $t()

Reply:
Enum not supports context.root.$t(). Need to handle it in different way, as these steps are using on button switch cases, we can handle it on validation ticket.